README.md

```diff
@@ -51,16 +50,22 @@ See also the [examples](examples).
 ### Server
 
 The `Server` class is responsible for handling incoming connections and then
-emit a `request` event for each incoming HTTP request.
+execute the execute the callback function passed to the constructor.
```

Comment:
Its responsibility is a bit unclear. How about just this:

…and then **processing each incoming HTTP request.** (or similar)